### PR TITLE
add EFs for electricity endpoint computing

### PR DESCRIPTION
**Pull request by gpantelidou**
**Is this a new addition or an update to an existing factor?**

- [X] New
- [ ] Update

**What category or categories does it affect?**

Electricity

**What?**

add missing EFs for electricity endpoint computing 

**Why?**

useful EFs for the endpoint computing project and now new data in the OEFDB

### CODE REVIEW

**Comment by gpantelidou** [comment]
The EFs reported under AR5 have been checked! 
I didn't include the text "not-supplied" to be consistent with data from previous years but I will look again.

**Comment by patstirling** [comment]
Are we sure about Google being lca_activity=`electricity_generation` not `electricity_consumption` - e.g. does it include t&d

**Comment by gpantelidou** [comment]
@patstirling About Google lca_activity:  I changed it to electricity_generation after this discussion on linear: 
https://linear.app/climatiq/issue/PROD-633/lca-activities-of-emission-factors

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -21651,3 +21651,72 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
 dcd10ed9-4759-4921-8a69-52d7a869820c,Energy,Fuel,fuel_type_natural_gas-fuel_use_residential_construction_commercial_institutional_agriculture,Natural gas (residential/construction/commercial/institutional/agriculture subsector),m3,not-supplied,not-supplied,1.939,3.7e-05,3.5e-05,not-supplied,not-supplied,,,unknown,UNFCCC,National Inventory Report,2017,2015,,CA-YT,,Climatiq,2022/12/13,Emission intensity of natural gas (marketable) consumed by residential/construction/commercial/institutional/agriculture subsector. Published in Canada 2022 National Inventory Report (NIR).,https://unfccc.int/documents/461919
+UUID,sector,category,activity_id,name,activity_unit,kgCO2e-AR5,kgCO2e-AR4,kgCO2,kgCH4,kgN2O,kgCO2e-OtherGHGs-AR5,kgCO2e-OtherGHGs-AR4,uncertainty,scope,lca_activity,source,dataset,year_released,year_valid_from,years_calculated_from,region,data_quality,contributor,date_accessed,description,source_link
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:21654]: Everything looks clean to me @gpantelidou - just in this line is a copied header, should be deleted :)

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (gpantelidou) [reply OpenEmissionFactorsDB.csv:21654]: Oh no, ah thanks for spotting that !!!

**Pull request closed without merging**